### PR TITLE
LMSDEMO-78 - Downloads

### PR DESCRIPTION
**Pull request by pieecia**

### CODE REVIEW

**Review by qunabu — CHANGES_REQUESTED** [review]
please describe grid in advance topics

File: doc/advanced/grid.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+`react-grid-system` provides a responsive grid for React inspired by Bootstrap. Moreover, it has various additional powerful features, such as setting breakpoints and gutter widths through React's context.
+
```
Comment (qunabu) [inline doc/advanced/grid.md:2]:
```suggestion
```js 
import { Container, Row, Col } from 'react-grid-system';

<Container>
  <Row>
    <Col sm={4}>
      One of three columns
    </Col>
    <Col sm={4}>
      One of three columns
    </Col>
    <Col sm={4}>
      One of three columns
    </Col>
  </Row>
</Container>
```
```

File: doc/advanced/grid.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+`react-grid-system` provides a responsive grid for React inspired by Bootstrap. Moreover, it has various additional powerful features, such as setting breakpoints and gutter widths through React's context.
```
Comment (qunabu) [inline doc/advanced/grid.md:1]:
```suggestion
[`react-grid-system`](https://www.npmjs.com/package/react-grid-system
) provides a responsive grid for React inspired by Bootstrap. Moreover, it has various additional powerful features, such as setting breakpoints and gutter widths through React's context.
```

File: doc/advanced/grid.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+https://www.npmjs.com/package/react-grid-system
```
Comment (qunabu) [inline doc/advanced/grid.md:3]:
```suggestion
```

**Review by qunabu — CHANGES_REQUESTED** [review]

File: src/components/molecules/Downloads/Downloads.md [hunk 4]
```diff
@@ -6,10 +6,23 @@ import themes from "../../../theme";
...
+    <Downloads
+      title={"Downloads"}
+      subtitle={"Subtitle"}
+      downloads={[
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+      ]}
```
Comment (qunabu) [inline src/components/molecules/Downloads/Downloads.md:23]: what i thought originaly is that we pass `array` of **Data** (DTO) and this component will generate  map of ` <Download />` as a result ...

**Review by qunabu — CHANGES_REQUESTED** [review]

File: src/components/molecules/Downloads/Downloads.md (hunk 4 above)
Comment (pieecia) [reply src/components/molecules/Downloads/Downloads.md:23]: ok, so the structure should be like this?

```
downloads={[
      {
          fileName: "File name",
          href: "https://www.google.com",
      },
      {
          fileName: "File name",
          href: "https://www.google.com",
      }
  ]}
```

Comment (qunabu) [reply src/components/molecules/Downloads/Downloads.md:23]: yep

**Review by qunabu — APPROVED** [review]

**Pull request merged**